Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # Support FROM override
-ARG BUILD_IMAGE=docker.io/golang:1.20.7@sha256:741d6f9bcab778441efe05c8e4369d4f8ff56c9a635a97d77f55d8b0ec62f907
+ARG BUILD_IMAGE=docker.io/golang:1.20.8@sha256:1b46d1164aceec47750cc0c3a6f8c44ee8050a0745c07affcef73518544a2b1f
```

Comment:
Where did you get the digest? If I pull golang:1.20.8 I get sha `6b29720c5598cb40b23dd38f522050a698d427c3a5696c5efe1ed22bd45b1396` ? 

Did you pull arm64 on Mac?